operators/deployment-validation-operator/0.2.2/manifests/deploymentvalidationoperator.0.2.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: Application Runtime, Monitoring, Security
+    certified: "false"
+    containerImage: quay.io/deployment-validation-operator/dv-operator:0.2.2
+    createdAt: 12/14/2021
+    description: The deployment validation operator
+    repository: https://github.com/app-sre/deployment-validation-operator
+    support: Best Effort
+  name: deployment-validation-operator.v0.2.2
+  annotations:
+    ignore-check.kube-linter.io/minimum-three-replicas: "This deployment uses 1 pod as currently replicating does not replicate metric data causing installation issues"
+spec:
+  description: "The Deployment Validator Operator(DVO) checks deployments and other\
+    \ resources against a curated collection of best practices.\nThese best practices\
+    \ focus mainly on ensuring that the applications are fault-tolerant.\n\n### Configuration\n\
+    \ To configure the checks Operator runs or to run your own custom checks, you can use a \
+    \ ConfigMap created in the **same namespace** as operator itself. ConfigMap should be named as \
+    \ *deployment-validation-operator-config* and contain *deployment-validation-operator-config.yaml* as data.\
+    \ [Example](https://github.com/app-sre/deployment-validation-operator/blob/master/deploy/openshift/configmap.yaml)\n\
+    \ \n### Metrics\nDVO will report failed validations via Prometheus metrics. All the metrics\
+    \ are gauges that will report `1` if the best-practice has failed.\nThere is \
+    \ [Instruction](https://github.com/app-sre/deployment-validation-operator#install-dashboard)\
+    \ to install a simple grafana dashboard."
+  displayName: Deployment Validation Operator
+  replaces: deployment-validation-operator.v0.2.1
```

Comment:
@npecka Please remove `replaces:` because you are not replacing any version since there is only `0.2.2`